src/nspawn/nspawn.c

```diff
@@ -188,6 +188,7 @@ static char **arg_network_veth_extra = NULL;
 static char *arg_network_bridge = NULL;
 static char *arg_network_zone = NULL;
 static char *arg_network_namespace_path = NULL;
+static const char *arg_network_provided_mac = NULL;
```

Comment:
we generally prefer to store already parsed fields here, hence please change type from `const char*` to `struct hw_addr_data` or so.

Reply:
Suggestion applied.